README.md

```diff
@@ -312,7 +312,7 @@ To generate a troubleshooting log for the LSP server, specify a value for the pa
 
 These values are ordered from returning the least information ("ERROR"; errors only) to the most information ("ALL"; all details).
 
-Troubleshooting logs are stored in the subfolder **\LSPCobol\logs** within your user home directory.
+Server logs could be found under "COBOL Language Support" output terminal.
```

Comment:
@zeibura Please check the update in README

Reply:
Should be called either just "output" or "output channel" according to VS Code terminology.